source/core/parameters_lagrangian.cc

```diff
@@ -1395,6 +1395,11 @@ namespace Parameters
           "false",
           Patterns::Bool(),
           "State whether Lagrangian post-processing should be performed.");
+        prm.declare_entry(
+          "force chains post-processing",
```

Comment:
no need for the post-processing because this is already in a post-processing section
I would just write force chains